frontend/app/src/components/home/profile/ReceiveCrypto.svelte

```diff
@@ -44,7 +46,11 @@
             on:error={onBalanceRefreshError} />
     </span>
     <form class="body" slot="body">
-        <AccountInfo qrSize={"larger"} centered {ledger} user={$user} />
+        {#if symbol === BTC_SYMBOL}
```

Comment:
Currently, the symbol will never be BTC, because we're still using ckBTC.
So this change is completely safe to get in.
I'll make a subsequent change that switches ckBTC to BTC and then we can test everything end to end on prod test.